src/components/sponsors/index.js

```diff
@@ -0,0 +1,79 @@
+import styles from './style.less';
+
+/**
+ * Sponsors on the main page is a unique selling point of our
+ * gold and platinum tier on opencollective. See:
+ * https://opencollective.com/preact for an overview of
+ * available tiers and their advantages.
+ */
+export default function Sponsors() {
+	return (
+		<div>
+			<ul class={styles.sponsorList}>
+				<li class={styles.sponsorItem}>
+					<a href="https://amp.dev/" title="AMP">
+						<AMPLogo />
+					</a>
+				</li>
+				<li class={styles.sponsorItem}>
+					<a href="https://webflow.com/" title="webflow">
+						<WebflowLogo />
+					</a>
+				</li>
+			</ul>
+		</div>
+	);
+}
+
+function AMPLogo() {
```

Comment:
Any chance we could use `<img src="logo.svg">` rather than inline SVG here? It'd keep DOM size down and bundles small.

Reply:
Sure thing, switched to external svg files with the next commit :+1: